.infrastructure/deployment.yml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+ name: kube2py
+ namespace: mateapp
+spec:
+ replicas: 2
+ selector:
+   matchLabels:
+     app: kube2py
+ strategy:
+   rollingUpdate:
+     maxUnavailable: 0
```

Comment:
Setting `maxUnavailable` to `0` can lead to deployment issues, as it doesn't allow any pods to be unavailable during updates. Consider setting it to `1` or a percentage value to ensure smoother rollouts.